requirements.txt

```diff
@@ -5,13 +5,13 @@ asn1crypto==1.5.1
 async-timeout==4.0.2
 attrs==22.2.0
 Authlib==1.2.0
-awscli==1.27.94
+awscli==1.27.95
 bcrypt==4.0.1
 billiard==3.6.4.0
 bleach==6.0.0
 blinker==1.5
-boto3==1.26.94
-botocore==1.29.94
+boto3==1.26.95
+botocore==1.29.95
```

Comment:
These are top-level requirements.  Although not strictly necessary, whenever I see top-level updates in requirements.txt, I also update them in requirements-app.txt.  See [step 4](https://github.com/department-of-veterans-affairs/notification-api#update-requirementstxt).

Reply:
Added to requirements-app.txt.